app/src/main/java/pl/school/register/model/Mark.java

```diff
@@ -11,12 +12,20 @@ public class Mark {
     @ManyToOne
     private Student student;
 
+    @ManyToOne
+    private Teacher teacher;
```

Comment:
We can get this information; 
mark -> meeting _id-> lesson_id -> teacher_id

However, since its pretty heavy i think its ok to add that